Library/Formula/thewidgetfactory.rb

```diff
@@ -0,0 +1,66 @@
+class Thewidgetfactory < Formula
+  desc "The Widget Factory is a showcase tool for GTK+ widgets"
+  homepage "http://www.stellingwerff.com/?page_id=10"
+  url "http://www.stellingwerff.com/TheWidgetFactory/thewidgetfactory-0.2.1.tar.gz"
+  sha256 "10bbbe50dc23a0cd47762749c6bb402559721951f3c6af40366258593bcb1ce4"
+
+  depends_on "pkg-config" => :build
+  depends_on "gtk+"
+
+  patch :DATA
```

Comment:
OK, so you would rather have it ugly (gtk+ and twf) than doing any patching downstream ?
I suppose that "works", but it does look a little grey and it does output some deprecations.